docs/standard/using-linq.md

```diff
@@ -168,7 +168,7 @@ public class DogHairLengthComparer : IEqualityComparer<Dog>
     public int GetHashCode(Dog d)
     {
         // default hashcode is enough here, as these are simple objects.
-        return b.GetHashCode();
+        return d.GetHashCode();
```

Comment:
I think this code is still wrong, but I also think that's outside the scope of this Pull Request, so I opened a separate issue about that: https://github.com/dotnet/docs/issues/7002.

Reply:
Yeah, let's keep this one focused on the typo